aspnet/hosting/index.rst

```diff
@@ -0,0 +1,13 @@
+Hosting
+=======
+
+.. toctree::
+   :titlesonly:
+   
+   servicing
```

Comment:
@blowdart Does this seem more reasonable https://github.com/shirhatti/Docs/commit/f251c6d605ce6ddddf360713f0f0589c4440b9bb ?